scripts/e2e/us-dump.sh

```diff
@@ -40,8 +40,9 @@ if [ $CONNECTION_CHECK_RESULT_STATUS_CODE -ne 0 ]; then
     exit 1
 fi
 
-DUMP_CMD="pg_dump --dbname=$US_CONNECTION --column-inserts -a -t workbooks -t entries -t revisions -t links"
-docker exec -it $US_CONTAINER_ID /bin/sh -c "$DUMP_CMD" | grep -Ev "^(--|SET|SELECT pg_catalog.set_config)" >$DUMP_TMP
+DUMP_CMD="pg_dump --dbname=$US_CONNECTION --disable-triggers --column-inserts -a -t collections -t workbooks -t entries -t revisions -t links"
+docker exec -it $US_CONTAINER_ID /bin/sh -c "$DUMP_CMD" |
+tee >(grep -Ev "^(--|SET|SELECT pg_catalog.set_config|pg_dump:)" >$DUMP_TMP) | grep "pg_dump:"
```

Comment:
After adding the collections table to the dump, I started getting pg_dump warnings in the dump file due to a circular foreign key in the collections table.

pg_dump: warning: there are circular foreign-key constraints on this table:
pg_dump: detail: collections
pg_dump: hint: You might not be able to restore the dump without using --disable-triggers or temporarily dropping the constraints.
pg_dump: hint: Consider using a full dump instead of a --data-only dump to avoid this problem.
I added ```--disable-triggers``` flag to pg_dump to solve problem if we will have collections with circular dependencies in the future
But warnings were still written to the file
To resolve this, I filtered the warnings from the dump file and redirected the warnings output to the console.